src/Cli/dotnet/Parser.cs

```diff
@@ -160,9 +160,20 @@ internal class CommandLineConsole : IConsole
 
         internal class DotnetHelpBuilder : HelpBuilder
         {
-            public DotnetHelpBuilder(IConsole console) : base(console) { }
+            public DotnetHelpBuilder(IConsole console, int maxWidth = int.MaxValue) : base(console, maxWidth) { }
 
-            public static Lazy<HelpBuilder> Instance = new Lazy<HelpBuilder>(() => new DotnetHelpBuilder(new CommandLineConsole()));
+            public static Lazy<HelpBuilder> Instance = new Lazy<HelpBuilder>(() => {
+                int windowWidth;
+                try
+                {
+                    windowWidth = System.Console.WindowWidth;
+                }
+                catch
+                {
+                    windowWidth = int.MaxValue;
```

Comment:
This is the recommended value? That's an awfully wide window

Reply:
Yes, that's the default value system command line uses.